interfaces/cython/cantera/kinetics.pyx

```diff
@@ -365,6 +365,26 @@ cdef class Kinetics(_SolutionBase):
         def __get__(self):
             return get_reaction_array(self, kin_getDeltaSSEntropy)
 
+    property heat_release_rate:
+        """
+        Get the total volumetric heat release rate [W/m^3].
+        """
+        def __get__(self):
+            return - np.sum(self.partial_molar_enthalpies *
+                            self.net_production_rates, 0)
+
+    property heat_production_rates:
+        """
+        Get the volumetric heat production rates [W/m^3] on a per-reaction
+        basis. The sum over all reactions results in the total volumetric heat
+        release rate.
+        Example: C. K. Law: Combustion Physics (2006), Fig. 7.8.6
+
+        >>> gas.heat_production_rates[1]  # heat production rate of the 2nd reaction
+        """
+        def __get__(self):
+            return - self.net_rates_of_progress * self.delta_standard_enthalpy
```

Comment:
Shouldn't this use `delta_enthalpy` instead of `delta_standard_enthalpy`? They're the same in the ideal gas case, but otherwise I think this would be inconsistent with the implementation o `heat_release_rate`.

Reply:
This was based on a simple cut&paste job :man_shrugging: ... I'm used to gas phase chemistry where I typically define everything based on standard enthalpies, so what's there is consistent with my understanding (see also my unit test). Of course, I may be missing something as not everything is gas phase ...

Reply:
Be it as it may, I changed it ...

Reply:
In the case of a real gas, using the "standard" enthalpy (per Cantera's definition, however problematic it may be) would miss any impact of the interaction terms. So from a consistency standpoint, I think you want to also use the full version of the enthalpy (`partial_molar_enthalpies`) in your test case.

Reply:
Done.